tests/nat/cli/commands/test_workflow_commands.py

```diff
@@ -12,6 +12,29 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+from click.testing import CliRunner
+from nat.cli.commands.workflow.workflow_commands import create_command
+
+def test_create_workflow_with_valid_name(tmp_path):
+    """Ensure CLI succeeds with a valid workflow name."""
+    runner = CliRunner()
+    result = runner.invoke(
+        create_command,
+        ["my-workflow", "--no-install", "--workflow-dir", str(tmp_path)]
+    )
+    assert result.exit_code == 0
+    assert "Workflow 'my-workflow' created successfully in" in result.output
+
+
+    # Verify the workflow directory was actually created
+    workflow_dir = tmp_path / "my-workflow"
+    assert workflow_dir.is_dir()
+
+    # Cleanup created workflow
+    if workflow_dir.exists():
+        for child in workflow_dir.iterdir():
+            child.unlink()
+        workflow_dir.rmdir()
```

Comment:
Please move below all other imports.

All imports need to be grouped together:

```python
import subprocess
from pathlib import Path
from unittest.mock import patch

import pytest
from click.testing import CliRunner

from nat.cli.commands.workflow.workflow_commands import create_command
from nat.cli.commands.workflow.workflow_commands import _get_nat_dependency
from nat.cli.commands.workflow.workflow_commands import get_repo_root
```